notebooks/product_kernel.py

```diff
@@ -0,0 +1,65 @@
+# %%
+%load_ext autoreload
+%autoreload 2
+# %%
+import lab.numpy as B
+import matplotlib.pyplot as plt
+from geometric_kernels.lab_extras.extras import from_numpy
+import numpy as np
+# %%
+
+from geometric_kernels.spaces import Circle, ProductDiscreteSpectrumSpace
+
+circle = Circle()
+
+# %%
+product = ProductDiscreteSpectrumSpace(circle, circle, num_eigen=11**2)
```

Comment:
I would expect this to be safer:
```suggestion
product = ProductDiscreteSpectrumSpace(Circle(), Circle(), num_eigen=11**2)
```